mdn/src/main/java/org/apache/james/mdn/MDNReportParser.java

```diff
@@ -0,0 +1,484 @@
+package org.apache.james.mdn;
+
+import java.util.Optional;
+
+import org.parboiled.BaseParser;
+import org.parboiled.Rule;
+
+import com.google.common.annotations.VisibleForTesting;
+
+public class MDNReportParser {
+    public MDNReportParser() {
+    }
+
+    public Optional<MDNReport> parse(String mdnReport) {
+        return Optional.empty();
+    }
+
+    @VisibleForTesting
+    static class Parser extends BaseParser<MDNReport> {
+        //   CFWS            =   (1*([FWS] comment) [FWS]) / FWS
```

Comment:
the comments are useful but the whitespaces are bothering

Reply:
I prefer to keep the original indent of the rfc